### PR TITLE
Implementing rate-limit override

### PR DESCRIPTION
**Pull request by Yoric**
This will be useful for https://github.com/matrix-org/mjolnir/pull/178/files/d2de4568526c94d50e0a8986dc3724f3929a2793#diff-a994e9304f29ad4a4427eeae61de99d9ec12907706637b9af53a5800ce0c1491 .

### CODE REVIEW

**Comment by Yoric** [comment]
Actually, I realized that it won't be helpful for that specific example, as Mjölnir tests create their users themselves, but it will still be useful for e.g. minotaur.

**Comment by Yoric** [comment]
While this patch is not necessary for Mjölnir, I realize that it has the side-effect of making the Synapse-won't-startup error a bit clearer.

File: src/registration.rs [hunk 1]
```diff
@@ -278,6 +300,17 @@ async fn ensure_user_exists(
 }
 
 pub async fn handle_user_registration(config: &crate::Config) -> Result<(), Error> {
+    // Create an admin user, we'll need it later.
```
Comment (Gnuxie) [inline src/registration.rs:303]: later for what?

**Review by Gnuxie — APPROVED** [review]
Looks good, it seems to work too

File: src/registration.rs (hunk 1 above)
Comment (Yoric) [reply src/registration.rs]: To unthrottle users.

Amended comment :)

**Pull request merged**